src/handler-context/context-builders/index.ts

```diff
@@ -0,0 +1,36 @@
+import { S3Event, SQSEvent } from "aws-lambda";
+import { CtxBuilderOptions, HandlerCtx } from "..";
+import { ConfigFileNames } from "../config/types";
+import { getBlankCtx } from "./blank";
+import { addConfigToCtx } from "./config";
+import { addEnvToCtx } from "./env";
+import { addLoggerToCtx } from "./logger";
+import { addMessagesToCtx } from "./messages";
+import { addOutputsToCtx } from "./outputs";
+
+export const buildContext = async <
+  TMessage,
+  TEnvVars extends string,
+  TConfigFileNames extends ConfigFileNames
+>(
+  event: S3Event | SQSEvent,
+  {
+    envVars,
+    messageTypeGuard,
+    outputs,
+    configFiles,
+  }: CtxBuilderOptions<TMessage, TEnvVars, TConfigFileNames>
+): Promise<HandlerCtx<TMessage, TEnvVars, TConfigFileNames>> => {
+  // for the love of god find a nicer way to do this chain
```

Comment:
Put a TODO here so that SonarCloud can appropriately harass you. :-)

Reply:
Luckily thanks to @gdsgj I removed this comment altogether 😂